packages/cli-plugin-metro/src/commands/start/watchMode.ts

```diff
@@ -3,6 +3,11 @@ import {logger, hookStdout} from '@react-native-community/cli-tools';
 import execa from 'execa';
 import chalk from 'chalk';
 import {Config} from '@react-native-community/cli-types';
+import {KeyPressHandler} from '../../tools/KeyPressHandler';
+import {addInteractionListener} from '@react-native-community/cli-tools';
```

Comment:
nit: Merge with existing `'@react-native-community/cli-tools'` import.

Reply:
yeah, I will do it after https://github.com/react-native-community/cli/pull/2041 will be merged 👍